components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/GitpodGatewayClientCustomizationProvider.kt

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License-AGPL.txt in the project root for license information.
+
+package io.gitpod.jetbrains.remote
+
+import com.jetbrains.rdserver.unattendedHost.customization.GatewayClientCustomizationProvider
+import com.jetbrains.rdserver.unattendedHost.customization.controlCenter.DefaultGatewayControlCenterProvider
+import com.jetbrains.rdserver.unattendedHost.customization.controlCenter.GatewayControlCenterProvider
+import com.jetbrains.rdserver.unattendedHost.customization.controlCenter.GatewayHostnameDisplayKind
+import io.gitpod.jetbrains.remote.icons.GitpodIcons
+import javax.swing.Icon
+
+class GitpodGatewayClientCustomizationProvider : GatewayClientCustomizationProvider {
+    override val icon: Icon = GitpodIcons.Logo
```

Comment:
where can I see this icon?

Reply:
I thought it was not displayed to me because it's overwritten by the Alert Icon due to high ping.
So even with low ping you weren't able to see Gitpod logo there? (Should be displayed on the left side of the gitpod workspace name.)

Reply:
Could you check with JB folks please? 🙏

Reply:
Confirmed with JB that warning/danger icons override other icons.